tests/edges/test_core_edge_branches.py

```
@@ -279,10 +279,10 @@ async def test_unified_db_off_exception_and_invalid_ids(
     db = UnifiedFoodDatabase(cache_dir=str(tmp_path))
 
     class OffMock:
-        async def search_products(self, *args: Any, **kwargs: Any):  # noqa: D401
+        async def search_products(self, *_args: Any, **_kwargs: Any):  # noqa: D401
```

Ruff (0.13.3)

Unused `noqa` directive (non-enabled: `D401`)

Remove unused `noqa` directive

(RUF100)